sparkpost/django/email_backend.py

```diff
@@ -55,7 +57,7 @@ def _send(self, message):
                         'Content type %s is not supported' % alternative[1]
                     )
 
-        return self.client.transmissions.send(**params)
+        return self.client.transmissions.send(**params, **options)
```

Comment:
You can't pass two sets of kwargs to a method so you'll want to do a `params.update(options)` and then just pass ``**params` to the `send` method.